src/plugins/media/ui/ResizableMediaSingle/index.tsx

```diff
@@ -70,6 +71,9 @@ export default class ResizableMediaSingle extends React.Component<
     if (viewMediaClientConfig) {
       await this.checkVideoFile(viewMediaClientConfig)
     }
+
+    this.props.pctWidth &&
```

Comment:
Without this, the component doesn't know the image has been resized and will always go back to the min width if we recenter the image. Strange behaviour but I figured this fix was simple enough.

Reply:
Maybe it's better to put that comment in the code itself?

Reply:
Agreed